### ISSUE TITLE
@ContentChildren() unable to find nested children of element

### ISSUE THREAD

**Issue by davidwalschots** [angular/angular#14380]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Usage of the `formArrayName` directive stops `@ContentChildren()` from finding children of the element onto which `formArrayName` was applied.

**Expected behavior**
`@ContentChildren()` reaches children of `formArrayName`.

**Minimal reproduction of the problem with instructions**

[This plunker](http://plnkr.co/edit/q9DREq88aeAXO645X8zK?p=preview) shows the problem. `<my-component>` is defined twice in the template, but the length of the found components is 1.

**Please tell us about your environment:**

* **Angular version:** 2.4.7

**Comment by DzmitryShylovich** [comment]
http://plnkr.co/edit/KAunUm4qQ8hJkqHDqUvI?p=preview

**Comment by davidwalschots** [comment]
Interesting. In 2.0.1 change log:
ContentChild descendants should be queried by default (0dc15eb), closes #11645

What's the reasoning for not applying the same behaviour on ContentChildren?

**Comment by kara** [comment]
@davidwalschots See explanation [here](https://github.com/angular/angular/issues/13002#issuecomment-263981202).

Closing as a duplicate.
